src/test/java/com/prowidesoftware/swift/io/parser/SwiftParserConsumeBlockLenientTest.java

```diff
@@ -319,7 +319,7 @@ public void testSimpleBlockConsumerBlock3_3_KnownToFail() throws IOException {
         assertEquals("00112233", b.getTagValue("108"));
         assertTrue(b.containsTag("4"));
         assertEquals("foobar", b.getTagValue("4"));
-        assertEquals(b.getUnparsedTextsSize(), new Integer(1));
```

Comment:
Seems unrelated.

Reply:
it is related to the migration to Java 11

Reply:
I see, to avoid the deprecation warning.
This change does not directly have to do with the migration Jakarta. I am much in favour of moving to Java 11 first, release, then migrate to Jakarta.